_sass/components/_footnotes.scss

```diff
@@ -31,19 +31,19 @@ body {
   ol {
     @include heading('para-sm');
 
-    font-weight: 200;
+    font-weight: 400;
```

Comment:
Should be [`$weight-book`](https://github.com/18F/doi-extractives-data/blob/bb6b321ed9d83493867696e905ae2461c0c2d115/_sass/_variables.scss#L8)